tests/validate-dirty-db.py

```diff
@@ -103,7 +103,7 @@ def runNodeosAndGetOutput(myTimeout=3):
         expectedRetCode=2
         if retCode != expectedRetCode:
             errorExit("Expected return code to be %d, but instead received %d." % (expectedRetCode, retCode))
-        db_dirty_msg="database dirty flag set"
+        db_dirty_msg="atabase dirty flag set"
```

Comment:
This looks like an accidental deletion

Reply:
It might look like it, but it isn't. 😳 I would actually like to get rid of all the display text checks and stick with something more programatic like return codes, etc.